data/categories.txt

```diff
@@ -21,7 +21,7 @@ category "ship"
 	"Drone"
 
 
-# Ships of these types are able to be carried by other ships.
+# Ships of these types can be carried by other ships.
```

Comment:
Not a typo. How it was written is fine.
```suggestion
# Ships of these types are able to be carried by other ships.
```

Reply:
Literally every similar other sentence in that file uses 'can'. It is more concise and understandable (in my opinion, at least). Yes, technically the original version may be correct but it still feels to be out of place. Maybe we should get a third opinion on this.